### PR TITLE
Header fix for Phoenix 0.13.1

### PR DESCRIPTION
**Pull request by kurbaitis**
I installed clean Phoenix version together with blaguth. No luck to make it work because of bad formatted header (the error said that capital letters are not allowed). I guess because of newer phoenix/plug.

### CODE REVIEW

File: mix.exs [hunk 1]
```diff
@@ -3,7 +3,7 @@ defmodule Blaguth.Mixfile do
 
   def project do
     [app: :blaguth,
-     version: "1.1.1",
```
Comment (lexmag) [inline mix.exs:6]: Could you not change version? I'll update it in the separate commit; please note, it will be `v1.1.2`.

**Comment by lexmag** [comment]
Good find. Thank you!

File: mix.exs (hunk 1 above)
Comment (lexmag) [inline mix.exs:6]: Err, `v1.2.0`, of course.

**Comment by kurbaitis** [comment]
Removed the version. No problem.

**Comment by lexmag** [comment]
:+1:

**Pull request merged**